htdocs/core/modules/propale/doc/pdf_azur.modules.php

```diff
@@ -1609,9 +1609,9 @@ protected function _pagehead(&$pdf, $object, $showaddress, $outputlangs)
 			$pdf->MultiCell(80, 4, $carac_emetteur, 0, 'L');
 
 
-			// If CUSTOMER contact defined, we use it
+			// If BILLING contact defined, we use it
 			$usecontact = false;
-			$arrayidcontact = $object->getIdContact('external', 'CUSTOMER');
+			$arrayidcontact = $object->getIdContact('external', 'BILLING');
```

Comment:
On proposal, we want the contact "responsible of the proposal".
The billing contact will be used for the invoice.